python/ray/dashboard/modules/reporter/tests/test_actors.py

```diff
@@ -95,6 +96,9 @@ def loop(self):
     actor_id = a._ray_actor_id.hex()
     a.loop.remote()
 
+    # wait for loop() to start
+    time.sleep(0.5)
```

Comment:
Let's not add `time.sleep` in tests. We used to do this but avoid it now. We should be able to use `wait_for_condition` instead of sleep.